src/components/accession2/view/DetailPanel.tsx

```diff
@@ -190,7 +190,7 @@ export default function DetailPanel(props: DetailPanelProps): JSX.Element {
             <Grid item xs={gridRightSide} sx={valueStyle}>
               {`${strings.COLLECTED_FROM}${numPlants === undefined ? '' : ' ' + numPlants}${
                 collectionSource && collectionSource !== 'Other' ? ' ' + collectionSource : ''
-              } ${isNotPlural ? strings.PLANT : strings.PLANTS}`}
+              } ${isNotPlural ? strings.PLANT : strings.PLANTS.toLowerCase()}`}
```

Comment:
It was simpler and less error prone to capitalize the string in the strings file and apply `.toLowerCase()` to this instance of `strings.plant`, rather than using `.charAt(0)` & `.slice(1)` and concatenating the other instance.

Reply:
Yes, I prefer it like this, much simpler 👍